docs/REFERENCE.md

```diff
@@ -5,6 +5,7 @@ __(More examples coming soon!)__
 ###In this document:
 
 1. [`Remote` options](REFERENCE.md#remote-options)
+  + [Orderbook remote](REFERENCE.md#remote-orderbook)
```

Comment:
This probably belongs in a new category called "Subscriptions" (number 4) and could be called "Orderbook Subscription".